src/ducks/balance/Balance.jsx

```diff
@@ -299,6 +306,11 @@ class Balance extends React.Component {
     const accounts = accountsCollection.data
     const groups = groupsCollection.data
     const settings = settingsCollection.data
+    const transactions = transactionsCollection.data
+
+    const balanceHistories = getBalanceHistories(accounts, transactions)
```

Comment:
Je me demande si faire ça à chaque render, c'est pas un peu violent ? Peut être qu'on pourrait au memoizer le résultat (si accounts/transaction ne changent pas on renvoie le même résultat ? reselect comes to mind)

Reply:
J'avais pensé à ça aussi : https://github.com/cozy/cozy-client/issues/62 , ca m'intéresserait d'avoir votre avis (je sais pas si je suis très clair).

Reply:
Est-ce que https://github.com/cozy/cozy-banks/pull/314/files#diff-e141f225cc9aa907e4ac654b35f221dbR29 est OK pour toi ?

Reply:
Yep